zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -0,0 +1,514 @@
+                rsp.usedSize = lp.usedSize
+                found = True
+                break
+
+        if not found:
+            raise Exception('cannot found logical pool[%s], you must create it manually' % cmd.logicalPoolName)
+
+        return jsonobject.dumps(rsp)
+
+    @replyerror
+    def deploy_client(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.8.2)

`http` may be undefined, or defined from star imports

(F405)